src/main/java/com/qinvent/hr/service/EmployeeServIMPL.java

```diff
@@ -10,22 +10,40 @@
 import com.qinvent.hr.repository.EmployeeRepo;
 import com.qinvent.hr.service.EmployeeServ;
 
+/**
+ * @EmployeeServImpl - Implementation class for EmployeeServ Interface
+ */
 @Service
 public class EmployeeServIMPL implements EmployeeServ {
 	@Autowired
 	private EmployeeRepo empRepo;
 
+	/**
+	 *
+	 * @param employee - Adds New Employee to the database
+	 * @return emp - an object of Employee class
+	 */
 	@Override
 	public Employee addEmployee(Employee employee) {
 		Employee emp = empRepo.save(employee);
 		return emp;
 	}
+
+	/**
+	 *
+	 * @param id - Employee Unique ID - Auto Generated for each Employee
+	 * @return String - an Opeeration Successful Message
+	 */
 	@Override
 	public String removeEmployee(int id) {
 		empRepo.deleteById(id);
 		return "Employe Id: "+id+" Sucessfully Deleted";
 	}
 
+	/**
+	 * This Method is used to Find a Particular Employee by @param id
```

Comment:
We follow java doc with the below format:
- Function description
- List of parameters
- Return
- Mention the exception can be thrown by the function, if any